coalib/bearlib/aspects/meta.py

```diff
@@ -72,3 +73,23 @@ class Sub(subcls, aspectbase, metaclass=aspectclass):
 
     def __repr__(cls):
         return '<%s %s>' % (type(cls).__name__, repr(cls.__qualname__))
+
+
+def issubaspect(aspect, subaspect):
+    """
+    This function checks whether or not ``subaspect`` is a subaspect of
+    ``aspect``.
+    """
+    if not isinstance(aspect, aspectclass):
+        raise TypeError(
+            "{} is not an aspectclass' instance".format(
+                type(aspect).__name__))
+    if not isinstance(subaspect, aspectclass):
```

Comment:
Since `issubaspect` should also support instances of aspectclasses as `subaspect` you also have to check if otherwise `isinstance(subaspect, aspectbase)`